app/buck2_client/src/commands/test.rs

```diff
@@ -157,6 +157,36 @@ If include patterns are present, regardless of whether exclude patterns are pres
     #[clap(long)]
     test_executor_stderr: Option<OutputDestinationArg>,
 
+    #[clap(
+        long,
+        group = "default-info",
+        help = "Also build default info (this is not the default)"
+    )]
+    build_default_info: bool,
+
+    #[allow(unused)]
+    #[clap(
+        long,
+        group = "default-info",
+        help = "Do not build default info (this is the default)"
+    )]
+    skip_default_info: bool,
```

Comment:
do we need this, seems unused?   not passing build_default_info would already do the default?

Reply:
See other thread.